nisystemlink/clients/dataframe/models/_column.py

```diff
@@ -0,0 +1,22 @@
+from typing import Dict, Optional
+
+from nisystemlink.clients.core._uplink._json_model import JsonModel
+
+from ._column_type import ColumnType
+from ._data_type import DataType
+
+
+class Column(JsonModel):
+    """Defines a single column in a table."""
+
+    name: str
+    """The column name, which must be unique across all columns in the table."""
+
+    data_type: DataType
+    """The data type of the column."""
+
+    column_type: Optional[ColumnType] = ColumnType.Normal
```

Comment:
If you set `column_type` to None, would the serializer pass `"columnType": null` in the JSON, or leave it out of the JSON? The service will reject null, but will default to normal if you leave it out.

I ask because we may want to remove the `Optional` part if None would map to null in the JSON. Or if there's a way to annotate the model such that the serializer will leave out the key if it's None, then we could set it to none by default and let the server choose the default value instead of the client. But I don't know best practice on letting the client choose the default vs. the server.

Reply:
This is a good catch. It's serializing it to `null` and throwing an error. I removed the `Optional` annotation. 

In general, letting the server choose defaults is probably the better approach, but I wanted the default here so that the `Column` class could be reused for both responses and requests. Also, it's very unlikely the default will ever change from Normal.